README.md

```diff
@@ -1,70 +1,75 @@
+# Calculator.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> A functional calculator build with React.
 
-## Available Scripts
+## Liver Version
 
-In the project directory, you can run:
+[Calculator](https://react-calculator-sigmasam.herokuapp.com/)
 
-### `yarn start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+This project was built using these technologies:
+- React
+- JavaScript
+- GitHub
+- Eslint
+- Stylelint
+- VScode
+- Jest
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Getting Started
 
-### `yarn test`
+To get a local copy up and running follow these simple example steps.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Prerequisites
 
-### `yarn build`
+- Node.js is required to install dependencies and run scripts via npm.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- A modern browser
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Clone this repository:
+Clone with SSH:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+`git@github.com:SigmaSam/calculator.git`
 
-### `yarn eject`
+Clone with HTTPS
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+`https://github.com/SigmaSam/calculator.git`
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Clone with GitHub CLI
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+`gh repo clone SigmaSam/calculator.git`
+      
+## Setup
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+Install dependencies:
 
-## Learn More
+`$ npm install`
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Start the local webserver:
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+`$ npm start` 
+    
+This will open the calculator at local webserver at http://localhost:3000/
```

Comment:
**Required**

- [ ] Kindly add instructions on how to run your test cases in the Readme file. It is not the best to assume that your users know how to run tests. Stating the commands exclusively will ensure that you're leaving no stone unturned.